cpmpy/solvers/pindakaas.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python
+#-*- coding:utf-8 -*-
+##
+## pindakaas.py
+##
+"""
+    Interface to Pindakaas (`pkl`) API
```

Comment:
pkl for me is for Pickle files mentally

Given the name Pindakaas, 'pdk' seems more natural to me?